dynamoose.d.ts

```diff
@@ -306,4 +320,32 @@ declare module "dynamoose" {
     get(fn: any): any;
     set(fn: any): any;
   }
+
+  /**
+   * Transaction
+   */
+  export interface ModelTransactionConstructor<DataSchema, KeySchema> {
```

Comment:
`Model.transaction` methods don't return promises or have callbacks.

Reply:
Reference line 222, 241 in `lib/index.js`.
first paramater `items` of `dynamoose.transaction` is used `await Promise.all(items);`